redis/cache/redispipebp/monitored.go

```diff
@@ -20,12 +20,14 @@ import (
 const (
 	promNamespace = "redispipebp"
 
+	labelName    = "redis_name"
```

Comment:
Slug vs. name?

Reply:
I don't feel strongly either way. In `MonitoredSync` api it's called `Name` so I kept it consistent with that, but I can rename the prometheus label to slug if that's preferred.

Reply:
I don't have a good model for when something is a name vs a slug, so... 🤷 

Other names that might work: name, tag, target, group, remote, id

Reply:
we need andrew's tie breaking vote here but he's OOO today 😅

Reply:
ok renamed to `redis_slug`. currently across baseplate.go code we have several usages of _slug prometheus label and zero usage of _name in prometheus labels.

Reply:
Here's what I said to @fishy when he asked about this:

> When I was making the some other tooling, I think there was both a name and slug, name was the “human readable” one and slug was for the “machine readable” one that was actually in the metrics